modules/react-native-status/desktop/rctstatus.h

```diff
@@ -58,13 +58,13 @@ class RCTStatus : public QObject, public ModuleInterface {
     Q_INVOKABLE static bool JSCEnabled();
     Q_INVOKABLE static void jailSignalEventCallback(const char* signal);
 
-    void emitSignalEvent(const char* signal);
+    void emitSignalEvent(QString signal);
 
 Q_SIGNALS:
-    void jailSignalEvent(const char* signal);
+    void jailSignalEvent(QString signal);
```

Comment:
Unrelated: I believe jails have been removed recently, so this shouldn't be necessary anymore.